ee/localserver/mocks/querier.go

```diff
@@ -0,0 +1,51 @@
+// Code generated by mockery v2.20.0. DO NOT EDIT.
```

Comment:
How was this generated? Should it be encoded in a `go:generate` line?

Reply:
it was generated with [mockery](https://github.com/vektra/mockery) a cli tool for creating mocks, to use go:generate we would have to add mockery as a dependency I think?

Reply:
No. Only if you want `go generate ./` to work 😆 

It's not in the Makefile, because one of the generate commands require `thrift`, and I think one requires `grpc`

I think we should add this here as a way of documenting how it happened. I'd add some comments about how to install it or whatever.

Reply:
added a go generate line and some comments in the generated querier struct